# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1437431512 14400
#      Mon Jul 20 18:31:52 2015 -0400
# Node ID f74ab65ee1bf33383e73f05cbbc780a3afc11a89
# Parent  131ce8cfaa80f4a9d7ba16003a58ee8a7ba5a7de
Restore installation of tests from template source files

* liboctave/module.mk (LIBOCTAVE_TST_SRC): Fix typo of
$(LIBOCTAVE_TEMPLATE_SRC).

diff --git a/liboctave/module.mk b/liboctave/module.mk
--- a/liboctave/module.mk
+++ b/liboctave/module.mk
@@ -88,17 +88,17 @@ liboctave_liboctave_la_LDFLAGS = \
 
 ## Rules to build test files
 
 LIBOCTAVE_TST_SRC = \
   $(liboctave_array_libarray_la_SOURCES) \
   $(liboctave_numeric_libnumeric_la_SOURCES) \
   $(liboctave_system_libsystem_la_SOURCES) \
   $(liboctave_util_libutil_la_SOURCES) \
-  $(TEMPLATE_SRC)
+  $(LIBOCTAVE_TEMPLATE_SRC)
 
 LIBOCTAVE_TST_FILES_SRC := $(shell $(top_srcdir)/build-aux/find-files-with-tests.sh "$(srcdir)" $(LIBOCTAVE_TST_SRC))
 
 LIBOCTAVE_TST_FILES := $(addsuffix -tst,$(LIBOCTAVE_TST_FILES_SRC))
 
 liboctavetestsdir := $(octtestsdir)/liboctave
 
 nobase_liboctavetests_DATA = $(LIBOCTAVE_TST_FILES)
